src/scss/attendee/theme.scss

```diff
@@ -2,18 +2,54 @@
 // Please make sure their names have semantic meaning
 // i.e. instead of $white it would be $c-button--fg or $c-input--bg
 
-$sparkle--bg: rgb(99, 42, 199); // previously named $sparkle-theme
-$sparkle--border: rgba(130, 130, 130, 0.3); // previously named $sparkle-border
+// All the colour definitions are dona via CSS variables and then we use SCSS
+// variables to substitue them into the right place.
+:root {
+  --default-text: rgb(0, 0, 0);
+  --chat-input-background: rgb(230, 230, 230);
+
+  --panel-background: rgb(255, 255, 255);
+  --glass-panel-background: rgba(255, 255, 255, 0.4);
+
+  --sparkle--bg: rgb(99, 42, 199);
+  --sparkle--border: rgba(130, 130, 130, 0.3);
+
+  --button-fg: rgb(255, 255, 255);
+  --button-bg: rgb(99, 42, 199);
+
+  --overlay-fg: rgb(255, 255, 255);
+  --overlay-bg: rgba(99, 42, 199, 0.7);
+
+  @media (prefers-color-scheme: dark) {
+    --default-text: rgb(255, 255, 255);
+    --chat-input-background: rgb(60, 60, 60);
+
+    --panel-background: rgb(0, 0, 0);
+    --glass-panel-background: rgba(0, 0, 0, 0.4);
+  }
+}
+
+// This is the colour that should be used for 90% of text
+$c-default-text: var(--default-text);
+// Used wherever we are putting text on to a light background
```

Comment:
... like here: what does it mean "text onto a light background" if you can flip light and dark mode?

Are some colors going to be invariant to light/dark?
Then the separation above will make it clear which of the 3 cases a color belongs to.